app/src/features/FhirAttributePanel/FhirAttributePanel.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+
+import { Button, makeStyles, Typography } from "@material-ui/core";
+import { Add } from "@material-ui/icons";
+import { useTranslation } from "react-i18next";
+
+import { useAppSelector } from "app/store";
+import AttributeInputGroup from "features/FhirAttributePanel/AttributeInputGroup";
+import { selectSelectedNode } from "features/FhirResourceTree/resourceTreeSlice";
+import {
+  useApiAttributesListQuery,
+  useApiInputGroupsListQuery,
+  useApiInputGroupsCreateMutation,
+} from "services/api/endpoints";
+
+const useStyles = makeStyles((theme) => ({
+  button: {
+    textTransform: "none",
+    marginTop: theme.spacing(2),
+  },
+  container: {
+    padding: theme.spacing(2),
+  },
+}));
+
+const FhirAttributePanel = (): JSX.Element => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+  const selectedNode = useAppSelector(selectSelectedNode);
```

Comment:
I think we must use the router for this, not the store, to record the current selected node. When a node is selected, the route would be `/sources/{sourceId}/mappings/{resourceId}/attributes/{attributeId}`. The idea is that we should be able to share a link to the attribute we're working on.